File: Pascal/Prog_exe_02.pas
Program Prog_exe_02 ;

Var
	base, altura:integer;

 Begin
 
 	writeln('Programa para o cálculo da área de um triânguo:');
 	writeln('Escreva o comprimento da base deste triángulo:');
 	readln(base);
 	
 	writeln('Escreva o comprimento da altura deste triângulo:');
 	readln(altura);
 	
 	writeln('A área deste triângulo é de: ', (0.5 * base * altura));
  
 End.

File: C/Prog_exe_02.c
#include <stdio.h>

int main(int argc, char const *argv[])
{

	int base, altura;

	printf("%s\n", "Programa para o cálculo da área de um triângulo:");
	printf("%s\n", "Escreva o comprimento da base deste triângulo:");
	scanf("%d", &base);
	printf("%s\n", "Escreva o comprimento da altura deste triângulo:");
	scanf("%d", &altura);

	printf("A área deste triângulo é de: %f\n", (0.5 * base * altura));

	return 0;
}